membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -40,4 +49,27 @@ class AttributeController {
     onSuccess = Features.fromAttributes,
     onNotFound = Features.unauthenticated
   )
+
+  def membershipDetails = paymentDetails("Membership")
+  def digitalPackDetails = paymentDetails("Digital Pack")
+
+  def paymentDetails(service: String) = TouchpointFromCookieAction.async { implicit request =>
+    authenticationService.userId.fold[Future[Result]](Future(cookiesRequired)){ userId =>
```

Comment:
This should be moved out of the controllers, and the services should be instantiated once per backend type